hal_st/stm32fxxx/FlashInternalHighCycleAreaStm.hpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2024 Koninklijke Philips N.V.
+#ifndef HAL_FLASH_INTERNAL_HIGH_CYCLE_AREA_STM_HPP
+#define HAL_FLASH_INTERNAL_HIGH_CYCLE_AREA_STM_HPP
+
+#include DEVICE_HEADER
+#include "hal/interfaces/Flash.hpp"
+#include "hal_st/cortex/InterruptCortex.hpp"
+#include "infra/util/MemoryRange.hpp"
+#include <cstdint>
+
+namespace hal
+{
+    class FlashInternalHighCycleAreaStm : public Flash
```

Comment:
```suggestion
    class FlashInternalHighCycleAreaStm
        : public Flash
```

Reply:
Okay, I thought the formatting rules would enforce this already.